docs/articles/documentation/test-api/selecting-page-elements/dom-node-state.md

```diff
@@ -1,19 +1,15 @@
 ---
 layout: docs
-title: DOM Node Snapshots
-permalink: /documentation/test-api/selecting-page-elements/dom-node-snapshots.html
-checked: true
+title: DOM Node State
+permalink: /documentation/test-api/selecting-page-elements/dom-node-state.html
+checked: false
 ---
-# DOM Node Snapshots
+# DOM Node State
 
-A *DOM node snapshot* is an object that represents the state of a particular document node server side.
-This node can be a DOM element, document, fragment, text or comment node. DOM node snapshots are returned by [selectors](selectors.md).
+Selectors allows to get a server-side representation of the DOM node's state
+from the webpage that exposes API that is similar to DOM objects.
```

Comment:
> I guess we need a note that this selector properties are async

There is a link 'how to obtain element state' in the next line. In the 'how to obtain..' article we write that these properties are async. 
I think we should not write about it right here because here we just describe all available methods (they are actual for selectors and snapshots both)